src/Service/Services/MetadataProviders/ISqlMetadataProvider.cs

```diff
@@ -38,6 +38,16 @@ public interface ISqlMetadataProvider
 
         (string, string) ParseSchemaAndDbTableName(string source);
 
+        /// <summary>
+        /// Obtains the underlying all column names for each entity name.
+        /// </summary>
+        List<string> GetSchemaGraphQLFieldNamesForEntityName(string entityName);
+
+        /// <summary>
+        /// Obtains the underlying column type for an entity field.
```

Comment:
By 'column type' do you mean the GraphQL object type that represents the entity? or do you mean the database data type of the column like int, string, guid etc.?

e.g. 
```graphql
type Person {
  name: String
  age: Int
  picture: Url
}
```
I see this functions input argument fieldname would be a field like 'name' above

What does the graphQLType argument represent?

Reply:
Yes, it's the GraphQL object type. The name is getting from ```FieldDefinitionNode``` parsed by hot chocolate.